src/toxcore/state_format/old.rs

```diff
@@ -151,6 +160,37 @@ pub struct NospamKeys {
 /// Number of bytes of serialized [`NospamKeys`](./struct.NospamKeys.html).
 pub const NOSPAMKEYSBYTES: usize = NOSPAMBYTES + PUBLICKEYBYTES + SECRETKEYBYTES;
 
+/** E.g.
```

Comment:
Usage & some non-standard properties held by `Default` implementation. I *think* that derived `Default` implementation would just provide `0`s rather than random numbers as it should.

With that being said it's not a *real* example, so if you think that it's not needed/misleading, I'll just remove `E.g.`.